lighthouse-core/test/results/sample_v2.json

```diff
@@ -4827,12 +4827,12 @@
               "path": "3,HTML,1,BODY,17,BUTTON",
               "selector": "body > button.small-button",
               "boundingRect": {
+                "top": 484,
```

Comment:
just noticed these (huge) differences :o need to investigate

Reply:
ok I had to remind myself that this bounding rect is only used for full page screenshot positioning, and the value will have no effect on tap target results

I reckon numbers are so different because of `deviceScaleFactor`

Running LH against dbw_tester and viewing report shows that the screenshot lines up just fine.

I updated the sample artifact for TapTargets, it was a little old.